lib/mailtrap/action_mailer/delivery_method.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'base64'
```

Comment:
looks like it's not needed in this file; i'd move it to the file where it's used.

Reply:
that's true. It was here as the whole conversion implementation was here. Will move out, thanks